composer.json

```diff
@@ -59,8 +59,25 @@
 		],
 		"check-complete-strict": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPress"
+		],
+		"test": [
+			"@lint",
+			"@check-cs",
+			"@run-tests",
+			"@check-complete-strict"
 		]
 	},
+	"scripts-descriptions": {
+		"lint": "Lint PHP files against parse errors.",
+		"lint-ci": "Lint PHP files against parse errors, but output the results as Checkstyle XML (used for CI purposes).",
+		"check-cs": "Run the PHPCS script against the entire codebase.",
+		"fix-cs": "Run the PHPCBF script to fix all the autofixable violations on the codebase.",
+		"install-codestandards": "Manually install the coding standards, so that they are discoverable by PHPCS script. Used mostly for CI purposes",
+		"run-tests": "Run all the unit tests for the WordPress Coding Standards sniffs.",
+		"check-complete": "Check if all the sniffs are feature complete (has unit tests and XML documentation), \n but don't warn if the documentation is missing for the sniffs, or if there are documentation and test files present that are not associated with a particular sniff.",
```

Comment:
Why force the line split? Why not let the CLI's settings choose where to wrap the line?

Reply:
I mean, I can leave it out, but it looks too big either way:

<img width="1740" alt="image" src="https://user-images.githubusercontent.com/8638515/204235165-3c6cec9f-3c7d-4ced-885a-90a3a0218eb8.png">

<img width="841" alt="image" src="https://user-images.githubusercontent.com/8638515/204235219-c87e2540-825b-4152-9a1b-c1d281569507.png">

Ideally I'd like to shorten this, but not sure to what 🙈

Reply:
Just saw the comment by Juliette, I'll apply the short explanation so this won't bother us 😄